README.md

```diff
@@ -1,50 +1,88 @@
-* For parallel port based cables: Linux 3.x and 2.6 variants. 2.0, 2.2 and 2.4 might still work, but have not been tested for ages. For Linux, i386 and AMD64 architectures are supported.
-* For parallel port based as well as USB based cables: Windows NT 4.0, 2000, XP and Server 2003, Vista, 7 and 8. For USB based cables, NT 4.0 is not supported, though. The i386 architecture a.k.a "x86" ("32 bit") is fully supported; additionally, 64 bit Windows ("x64", "x86_64") versions are supported. Itanium-based Windows ("iA64") are not supported, though.
+* For USB based cables: Any Linux, FreeBSD or MacOS X variant that support
+  libusb-0.1 should be supported. Linux, FreeBSD and Mac OS X have been
+  explicitly tested.
+
+* For parallel port based cables: Linux 3.x and 2.6 variants. 2.0, 2.2 and 2.4
+  might still work, but have not been tested for ages. For Linux, i386 and
+  AMD64 architectures are supported.
+
+* For parallel port based as well as USB based cables: Windows NT 4.0, 2000, XP
+  and Server 2003, Vista, 7 and 8. For USB based cables, NT 4.0 is not
+  supported, though. The i386 architecture a.k.a "x86" ("32 bit") is fully
+  supported; additionally, 64 bit Windows ("x64", "x86_64") versions are
+  supported. Itanium-based Windows ("iA64") are not supported, though.
 
 ##  Supported CBM hardware
 
 Currently, opencbm supports the following CBM devices:
 
 * VIC 1541, VIC1540 (all variants, including clones)
+
 * VIC 1570, VIC 1571 (including the 1571CR and the 1571 inside of a C128DCR)
-* VIC 1581 (not with d64copy ( d64copy), not with cbmformat ( cbmformat) or cbmforng ( cbmforng))
+
+* VIC 1581 (not with d64copy ( d64copy), not with cbmformat ( cbmformat) or
+  cbmforng ( cbmforng))
+
 * other CBM IEC drives, printers, and compatibles (only with cbmctrl ( cbmctrl))
-* VIC 8250, 8050, 4040, 2031, SFD 1001, and possibly other IEEE drives with an IEC to IEEE converter (for example, IEC2IEEE from Jochen Adler, cf. http://www.nlq.de/, or with a ZoomFloppy extension that lets you use IEEE devices directly.
-* VIC 1530 (a.k.a. C2N) tape device, and VIC 1531 with an appropriate adapter; both are supported by ZoomTape, only.
+
+* VIC 8250, 8050, 4040, 2031, SFD 1001, and possibly other IEEE drives with an
+  IEC to IEEE converter (for example, IEC2IEEE from Jochen Adler, cf.
+  http://www.nlq.de/, or with a ZoomFloppy extension that lets you use IEEE
+  devices directly.
+
+* VIC 1530 (a.k.a. C2N) tape device, and VIC 1531 with an appropriate adapter;
+  both are supported by ZoomTape, only.
 
 ## Further reading
 
-The current manual, including installation instructions, can be read online at http://opencbm.trikaliotis.net/.
+The current manual, including installation instructions, can be read online at
+http://opencbm.trikaliotis.net/.
 
-A Doxygen output of the sources (for developers) (still work-in-progress) can be found at http://opencbm.trikaliotis.net/doxygen/.
```

Comment:
Why do you believe the Doxygen is empty? I have not written an "Introductory Overview", but despite that, it is there. Have a look at "Data Structures" and "Files".